src/containers/lock/containers/LockForm/index.js

```diff
@@ -147,6 +148,7 @@ class LockContainer extends React.Component {
                                                     <TermTableCell {...{ dashboard }}>
                                                         <Field
                                                             name="productId"
+                                                            data-testId={"selectLockProduct-" + product.id}
```

Comment:
capitalisation of `data-testid` is incorrect. not even sure why it works this way :)